main.nf

```diff
@@ -180,22 +171,14 @@ if(params.readPaths){
         .into { raw_reads_fastqc; raw_reads_trimgalore; raw_reads_mirtrace }
 }
 
-
 // Header log info
-log.info """=======================================================
-                                          ,--./,-.
-          ___     __   __   __   ___     /,-._.--~\'
-    |\\ | |__  __ /  ` /  \\ |__) |__         }  {
-    | \\| |       \\__, \\__/ |  \\ |___     \\`-._,-`-,
-                                          `._,._,\'
-
-nf-core/smrnaseq : Small RNA-Seq Best Practice v${params.version}
-======================================================="""
+log.info nfcoreHeader()
+if(workflow.revision) summary['Pipeline Release'] = workflow.revision
 def summary = [:]
 summary['Run Name']            = custom_runName ?: workflow.runName
 summary['Reads']               = params.reads
 summary['Genome']              = params.genome
-summary['Trim min length']     = params.length
+summary['Trim min length']     = params.min_length
```

Comment:
```suggestion
summary['Min Trimmed Length']     = params.min_length
```

Reply:
Make sure you add all of the relevant parameters that you have defined in the pipeline here too. Same goes for `usage.md`, and `nextflow.config`. Hopefully, the new `parameters.json` format will make this all much easier when its implemented. All manual for now...